pkg/schema/definitions/handler.go

```diff
@@ -223,7 +223,11 @@ func listGVKModels(models proto.Models, groups *metav1.APIGroupList, crdCache wa
 				Version: version.Name,
 				Kind:    crd.Spec.Names.Kind,
 			}
-			gvkToCRD[gvk] = version.Schema.OpenAPIV3Schema
+			if version.Schema != nil {
+				gvkToCRD[gvk] = version.Schema.OpenAPIV3Schema
+			} else {
+				gvkToCRD[gvk] = nil
```

Comment:
nit: this is a `map[schema.GroupVersionKind]*apiextv1.JSONSchemaProps`.  We don't iterate over its keys, and as far as I can tell it is only used below as `CRD:       gvkToCRD[*gvk]`. Since we're not iterating over the map, explicitly adding an entry with a nil pointer doesn't seem to make sense - the default value for a pointer is already nil.
```suggestion
```